frame/benchmarking/src/lib.rs

```diff
@@ -775,21 +779,36 @@ macro_rules! impl_benchmark {
 							// This will enable worst case scenario for reading from the database.
 							$crate::benchmarking::commit_db();
 
+							// Reset the read/write counter so we don't count operations in the setup process.
+							$crate::benchmarking::reset_read_write_count();
+
 							// Time the extrinsic logic.
 							frame_support::debug::trace!(target: "benchmark", "Start Benchmark: {:?} {:?}", name, component_value);
 							let start_extrinsic = $crate::benchmarking::current_time();
 							closure_to_benchmark()?;
 							let finish_extrinsic = $crate::benchmarking::current_time();
 							let elapsed_extrinsic = finish_extrinsic - start_extrinsic;
+							// Commit the changes to get proper write count
+							$crate::benchmarking::commit_db();
 							frame_support::debug::trace!(target: "benchmark", "End Benchmark: {} ns", elapsed_extrinsic);
+							let read_write_count = $crate::benchmarking::read_write_count();
+							frame_support::debug::trace!(target: "benchmark", "Read/Write Count {:?}", read_write_count);
```

Comment:
this logs also in wasm, which is probably not desirable I think? 

```suggestion

							frame_support::debug::native::trace!(target: "benchmark", "Read/Write Count {:?}", read_write_count);
```

Reply:
I also want these logs to emit during wasm execution, which is the standard way to run these benchmarks. Per conversation in "dumb questions", this should have no overhead when the log flag is not included